### PR TITLE
docs: add use cases

### PR DESCRIPTION
**Pull request by sehilyi**
Added the four use cases illustrated in the manuscript.

Screen Shot 2022-08-31 at 15 55 24

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -3,6 +3,6 @@
 * [Home](/)
 * [Quick Start](quick-start.md)
 * [Interact with Drava](user-interactions.md)
-* [Development Mode](dev.md)
 * [Use Cases](use-cases.md)
+* [Development Mode](dev.md)
```
Comment (sehilyi) [inline docs/_sidebar.md:7]: Making the interaction and use cases sections closer together seemed better.

File: docs/index.html [hunk 2]
```diff
@@ -15,7 +15,8 @@
       name: 'Drava',
       repo: 'https://github.com/wangqianwen0418/DRAVA',
       loadSidebar: true,
-      subMaxLevel: 3
+      subMaxLevel: 3,
+      themeColor: '#F7941F'
```
Comment (sehilyi) [inline docs/index.html:19]: Took an orange color from the interface & figures, but feel free to change the color.

File: docs/_sidebar.md (hunk 1 above)
Comment (wangqianwen0418) [reply docs/_sidebar.md:7]: Make sense to me! Thanks

**Review by wangqianwen0418 — APPROVED** [review]

File: docs/use-cases.md [hunk 3]
```diff
@@ -1,8 +1,155 @@
 # Use Cases
-Please refer to our [OSF reprint](https://osf.io/gzu27/) and [BioVis Presentation Video](https://drive.google.com/file/d/1-_5kkyez79Y2OvzOUrLeYJp0C-mv0XM6/view?usp=sharing) for more details about the use cases.
 
+?> This page illustrates the application scenarios of Drava.
 
-## Datasets Used
+<table>
```
Comment (wangqianwen0418) [inline docs/use-cases.md:5]: I add a gallery for all use cases

File: docs/use-cases.md [hunk 4]
```diff
@@ -1,8 +1,155 @@
...
...74(14)01497-4). The matrices describe the interactions between different genomic locations, which is related to the folding of DNA and affects regulation of gene expression. In a genome interaction matrix, rows and columns represent genomic locations and the color intensity indicates the interaction probability between a pair of locations. When visually exploring a large matrix, e.g.for 3M × 3M for human genome, experts typically examine regions of interest (ROI) that have unique visual patterns and indicate specific biological events. We generate small multiples for one specific type of ROIs called Topologically Associated Domains (TAD), which are visually represented as squares that are presumably organized hierarchically. We first extract TADs from the interaction matrix using [OnTAD](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1893-y) and then use the DRL model to generate a latent vector for each TAD. We demonstrate Drava using the 855 TADs extracted by OnTAD from chromosome 5 of the HFFc6 cell line. The DRL model has three convolution blocks with filter sizes of 7, 5, 3 and channel sizes of 32, 64, 128, respectively. The latent vector has 8 dimensions.
```
Comment (wangqianwen0418) [inline docs/use-cases.md:98]: I changed some of the citation styles

**Comment by wangqianwen0418** [comment]
@sehilyi, thanks so much for updating the docs!

I made some minor style changes and added a gallery for use cases.
image

Please take a look and see whether the changes make sense to you.
Otherwise, this PR is ready to merge.

File: docs/dev.md [hunk 5]
```diff
@@ -1,4 +1,6 @@
 # Run Drava in Development Mode
+?> This page illustrates how to run Drava in your local computer in the development mode.
+
 First, download or clone our github repo at https://github.com/wangqianwen0418/DRAVA.
```
Comment (sehilyi) [inline docs/dev.md:4]: Should we remove this link before submission?

File: docs/use-cases.md (hunk 3 above)
Comment (sehilyi) [reply docs/use-cases.md:5]: Nice!

File: docs/use-cases.md [hunk 6]
```diff
@@ -1,8 +1,155 @@
...
+
+This observation is valuable for understanding and improving this IDC diagnosis model. First, it indicates when and where the IDC prediction model tends to make confident false negative predictions and a double-check from human experts is needed. Second, the training strategy can be modified accordingly (e.g., increasing the sample weight of these loose and purple tissues) to improve the model performance.
+
+### Single Cell Masks
```
Comment (sehilyi) [inline docs/use-cases.md:133]: Just curious, is this also part of the manuscript?

**Comment by sehilyi** [comment]
Thanks for making further changes! They look great.

File: docs/dev.md (hunk 5 above)
Comment (wangqianwen0418) [reply docs/dev.md:4]: Good point. I have created another branch `anonymous` and will host the demo using that branch at https://anonymous.dcfg7f17k3km1.amplifyapp.com/ for the review process.

Comment (sehilyi) [reply docs/dev.md:4]: @wangqianwen0418 I think it is well-anonymized. Your name is shown in `package.json` and Google Docs, but I think that should be fine.

Screen Shot 2022-09-02 at 12 03 13

Screen Shot 2022-09-02 at 12 01 41

**Pull request merged**